src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -415,4 +418,20 @@ default boolean damage(double damage, DamageSource damageSource) {
         return getWorld().getEntities(predicate::test);
     }
 
+    /**
+     * Gets the current display name for this {@link Entity}.
+     *
+     * @return The display name
+     */
+    Text getDisplayName();
```

Comment:
All entities have a display name, and vanilla itself will break if it is not present/is null.